server/server.js

```diff
@@ -4,15 +4,18 @@ var webpack = require('webpack');
 var webpackMiddleware = require('webpack-dev-middleware');
 var config = require('../webpack.config.js');
 var mid = require('./middleware.js');
+var bodyParser = require('body-parser');
```

Comment:
move this up to line 3, so that it comes right after requiring express and right before requiring webpack